index.html

```diff
@@ -36,6 +38,13 @@
             if (vm.route.file !== 'README.md') html = html.replaceAll('Karen Bot', "[Karen Bot](https://karen.exerra.xyz)")
             return html
           })
+
+          // Adds the registered ASCII symbol to registered trademarks
+          hook.afterEach(html => {
```

Comment:
Not sure how I feel about hard-coding it in here since if you add more trademarks you will need to make PRs to add trademark regex and stuff, but otherwise good